[x264] Build failure on x64-windows

Package: x264[asm,core,gpl,tool]:x64-windows@0.164.3108#1

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.42.34435.0
-    vcpkg-tool version: 2024-12-09-1005b78fa1bf1dde1a20c2734cba4ea61ca94d9a
    vcpkg-scripts version: 80d54ff62 2024-12-23 (3 days ago)

**To Reproduce**

`vcpkg install --clean-buildtrees-after-build x264[asm,core,gpl,tool]:x64-windows`

**Failure logs**

```
-- Using cached videolan-x264-31e19f92f00c7003fa115047ce50978bc98c3a0d.tar.gz.
-- Extracting source K:/data/depot/git/vcpkg/downloads/videolan-x264-31e19f92f00c7003fa115047ce50978bc98c3a0d.tar.gz
-- Applying patch K:/data/depot/git/vcpkg/buildtrees/x264/src/version-0.164.3108.diff
-- Applying patch uwp-cflags.patch
-- Applying patch parallel-install.patch
-- Applying patch allow-clang-cl.patch
-- Applying patch configure.patch
-- Using source at K:/data/depot/git/vcpkg/buildtrees/x264/src/8bc98c3a0d-edb052ced6.clean
-- Getting CMake variables for x64-windows
-- Using cached msys2-autoconf-wrapper-20240607-1-any.pkg.tar.zst.
-- Using cached msys2-automake-wrapper-20240607-1-any.pkg.tar.zst.
-- Using cached msys2-binutils-2.43.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libtool-2.4.7-4-x86_64.pkg.tar.zst.
-- Using cached msys2-make-4.4.1-2-x86_64.pkg.tar.zst.
-- Using cached msys2-which-2.21-4-x86_64.pkg.tar.zst.
-- Using cached msys2-bash-5.2.037-1-x86_64.pkg.tar.zst.
-- Using cached msys2-coreutils-8.32-5-x86_64.pkg.tar.zst.
-- Using cached msys2-file-5.45-3-x86_64.pkg.tar.zst.
-- Using cached msys2-gawk-5.3.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-grep-1~3.0-6-x86_64.pkg.tar.zst.
-- Using cached msys2-gzip-1.13-1-x86_64.pkg.tar.zst.
-- Using cached msys2-diffutils-3.10-1-x86_64.pkg.tar.zst.
-- Using cached msys2-pkgconf-2.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-sed-4.9-1-x86_64.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.5.4-2-x86_64.pkg.tar.zst.
-- Using cached msys2-autoconf2.72-2.72-1-any.pkg.tar.zst.
-- Using cached msys2-automake1.16-1.16.5-1-any.pkg.tar.zst.
-- Using cached msys2-automake1.17-1.17-1-any.pkg.tar.zst.
-- Using cached msys2-libiconv-1.17-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libintl-0.22.4-1-x86_64.pkg.tar.zst.
-- Using cached msys2-zlib-1.3.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-tar-1.35-2-x86_64.pkg.tar.zst.
-- Using cached msys2-findutils-4.10.0-2-x86_64.pkg.tar.zst.
-- Using cached msys2-gmp-6.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-gcc-libs-13.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libbz2-1.0.8-4-x86_64.pkg.tar.zst.
-- Using cached msys2-liblzma-5.6.3-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libzstd-1.5.6-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libreadline-8.2.013-1-x86_64.pkg.tar.zst.
-- Using cached msys2-mpfr-4.2.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libpcre-8.45-4-x86_64.pkg.tar.zst.
-- Using cached msys2-m4-1.4.19-2-x86_64.pkg.tar.zst.
-- Using cached msys2-perl-5.38.2-2-x86_64.pkg.tar.zst.
-- Using cached msys2-ncurses-6.5.20240831-2-x86_64.pkg.tar.zst.
-- Using cached msys2-libxcrypt-4.4.36-1-x86_64.pkg.tar.zst.
-- Using msys root at K:/data/depot/git/vcpkg/downloads/tools/msys2/2db36fb050d01f45
-- Configuring x64-windows-rel
-- Configuring x64-windows-dbg
-- Using cached msys2-autoconf-wrapper-20240607-1-any.pkg.tar.zst.
-- Using cached msys2-automake-wrapper-20240607-1-any.pkg.tar.zst.
-- Using cached msys2-binutils-2.43.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libtool-2.4.7-4-x86_64.pkg.tar.zst.
-- Using cached msys2-make-4.4.1-2-x86_64.pkg.tar.zst.
-- Using cached msys2-which-2.21-4-x86_64.pkg.tar.zst.
-- Using cached msys2-bash-5.2.037-1-x86_64.pkg.tar.zst.
-- Using cached msys2-coreutils-8.32-5-x86_64.pkg.tar.zst.
-- Using cached msys2-file-5.45-3-x86_64.pkg.tar.zst.
-- Using cached msys2-gawk-5.3.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-grep-1~3.0-6-x86_64.pkg.tar.zst.
-- Using cached msys2-gzip-1.13-1-x86_64.pkg.tar.zst.
-- Using cached msys2-diffutils-3.10-1-x86_64.pkg.tar.zst.
-- Using cached msys2-pkgconf-2.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-sed-4.9-1-x86_64.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.5.4-2-x86_64.pkg.tar.zst.
-- Using cached msys2-autoconf2.72-2.72-1-any.pkg.tar.zst.
-- Using cached msys2-automake1.16-1.16.5-1-any.pkg.tar.zst.
-- Using cached msys2-automake1.17-1.17-1-any.pkg.tar.zst.
-- Using cached msys2-libiconv-1.17-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libintl-0.22.4-1-x86_64.pkg.tar.zst.
-- Using cached msys2-zlib-1.3.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-tar-1.35-2-x86_64.pkg.tar.zst.
-- Using cached msys2-findutils-4.10.0-2-x86_64.pkg.tar.zst.
-- Using cached msys2-gmp-6.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-gcc-libs-13.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libbz2-1.0.8-4-x86_64.pkg.tar.zst.
-- Using cached msys2-liblzma-5.6.3-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libzstd-1.5.6-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libreadline-8.2.013-1-x86_64.pkg.tar.zst.
-- Using cached msys2-mpfr-4.2.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libpcre-8.45-4-x86_64.pkg.tar.zst.
-- Using cached msys2-m4-1.4.19-2-x86_64.pkg.tar.zst.
-- Using cached msys2-perl-5.38.2-2-x86_64.pkg.tar.zst.
-- Using cached msys2-ncurses-6.5.20240831-2-x86_64.pkg.tar.zst.
-- Using cached msys2-libxcrypt-4.4.36-1-x86_64.pkg.tar.zst.
-- Using msys root at K:/data/depot/git/vcpkg/downloads/tools/msys2/2db36fb050d01f45
-- Building/Installing x64-windows-rel
-- Making target 'all' for x64-windows-rel
-- Making target 'install' for x64-windows-rel
-- Building/Installing x64-windows-dbg
-- Making target 'all' for x64-windows-dbg
-- Making target 'install' for x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_copy_tools.cmake:36 (message):
  Couldn't find tool "x264":

      "K:/data/depot/git/vcpkg/packages/x264_x64-windows/bin/x264.exe" does not exist
Call Stack (most recent call first):
  ports/x264/portfile.cmake:100 (vcpkg_copy_tools)
  scripts/ports.cmake:196 (include)



```

--- a/ports/x264/portfile.cmake
+++ b/ports/x264/portfile.cmake
@@ -89,9 +89,11 @@ vcpkg_make_configure(
     OPTIONS_RELEASE
         ${OPTIONS_RELEASE}
         --enable-strip
+        "--bindir=\\\${prefix}/bin"
     OPTIONS_DEBUG
         --enable-debug
         --disable-cli
+        "--bindir=\\\${prefix}/bin"
 )
 
 vcpkg_make_install()

--- a/ports/x264/vcpkg.json
+++ b/ports/x264/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "x264",
   "version": "0.164.3108",
-  "port-version": 1,
+  "port-version": 2,
   "description": "x264 is a free software library and application for encoding video streams into the H.264/MPEG-4 AVC compression format",
   "homepage": "https://www.videolan.org/developers/x264.html",
   "license": "GPL-2.0-or-later",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -9938,7 +9938,7 @@
     },
     "x264": {
       "baseline": "0.164.3108",
-      "port-version": 1
+      "port-version": 2
     },
     "x265": {
       "baseline": "4.1",

--- a/versions/x-/x264.json
+++ b/versions/x-/x264.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "9367314b9b01285a64c3c6b5e9f895d0ad0f0e48",
+      "version": "0.164.3108",
+      "port-version": 2
+    },
+    {
       "git-tree": "0b3254a0abf3a2a2748b7353594b401893f74da9",
       "version": "0.164.3108",
       "port-version": 1

